charts/redpanda/values.go

```diff
@@ -232,9 +232,10 @@ type PodTemplate struct {
 }
 
 type Statefulset struct {
-	NodeAffinity   map[string]any `json:"nodeAffinity"`
-	Replicas       int            `json:"replicas" jsonschema:"required"`
-	UpdateStrategy struct {
+	AdditionalSelectorLabels map[string]string `json:"additionalSelectorLabels"`
```

Comment:
Sorry, I don't recall if we talked about this already but is there any reason to _not` make this required (`jsonschema:"required"`)? It's nice to not have to worry about nulls when possible.

If we specify a default within `values.yaml` we won't require end users to make any changes as the JSONSchema is checked _after_ our `values.yaml` and the users' values are merged.

Reply:
not sure, but yeah we can make it required.